### PR TITLE
[14.0][IMP] product_import: always create product Active

### PR DESCRIPTION
**Pull request by florentx**

### CODE REVIEW

**Review by bosd — APPROVED** [review]
LGTM

**Review by mmequignon — APPROVED** [review]

File: product_import/wizard/product_import.py [hunk 1]
```diff
@@ -204,7 +204,11 @@ def create_product(self, parsed_product, chatter_msg, seller=None):
             product.write(product_vals)
             logger.info("Product %d updated", product.id)
         else:
+            product_active = product_vals.pop("active")
             product = self.env["product.product"].create(product_vals)
+            if not product_active:
+                # Product created first, then archived
+                product.action_archive()
```
Comment (sebalix) [inline product_import/wizard/product_import.py]: What is the purpose? A use case in the commit message or a comment is welcome.

Comment (florentx) [reply product_import/wizard/product_import.py]: I've added tests and comment.

**Review by sebalix — APPROVED** [review]

**Pull request merged**